src/data-display/dynamic/dynamic-layout/templates/raw-table/index.vue

```diff
@@ -74,6 +74,12 @@ export default defineComponent<RawTableDynamicLayoutProps>({
             fields: computed(() => {
                 if (state.rootData[0]) {
                     const firstItem = state.rootData[0];
+                    if (props.options?.headers) {
```

Comment:
suggestion: To ensure the headers option is array, what about to check the headers options is an array?

```suggestion
                    if (Array.isArray(props.options?.headers) {
```